lib/backends/wasm/ops/matmul.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {MatMul} from '../../../ops/matmul';
+import {Tensor} from '../../../tensor';
+import {BroadcastUtil, ShapeUtil} from '../../../util';
+import {WasmBinding} from '../../../wasm-binding';
+import {WasmInferenceHandler} from '../inference-handler';
+
+export class WasmMatMul extends MatMul {
+  run(inferenceHandler: WasmInferenceHandler, inputs: Tensor[]): Tensor[] {
+    const a = inputs[0];
+    const b = inputs[1];
+    let dimsA = a.dims.slice(0);
+    let dimsB = b.dims.slice(0);
+    // If the first argument is 1-D, it is promoted to a matrix by prepending
```

Comment:
according to the operator's [definition](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.matmul.html), the preprocess and postprocess should be used by all backends. is it a good idea to put in /lib/ops/matmul.ts

Reply:
Actually it is hard to put this in /lib/ops/matmul but it might make sense to put this into utils